packages/verification/src/models/v1/start-flashcall-verification-response/start-flashcall-verification-response.ts

```diff
@@ -5,7 +5,7 @@ export interface StartFlashCallVerificationResponse {
   /** Verification identifier used to query for status. */
   id?: string;
   /** The value of the method used for the Verification. For Flashcall Verifications, this will always be `flashCall`. */
-  method?: 'flashCall';
+  method?: 'flashcall';
```

Comment:
💡 Did you check against server, just to ensure there is not another mix here between `flashCall` and `flashcall` we had in the past